curriculum/challenges/english/blocks/workshop-envelope-budget-app/63c9bf80558d780e848b2987.md

```diff
@@ -0,0 +1,316 @@
+---
+id: 63c9bf80558d780e848b2987
+title: Step 70
+challengeType: 0
+dashedName: step-70
+---
+
+# --description--
+
+Using that same syntax, query your `number` inputs in the `#utilities` element and assign them to `utilitiesNumberInputs`.
+
+# --hints--
+
+You should declare a `utilitiesInputs` variable.
+
+```js
+assert.match(calculateBudget.toString(), /utilitiesInputs\s*=/);
+```
+
+You should call `document.querySelectorAll()` with `#utilities input[type='number']`.
+
+```js
+assert.match(calculateBudget.toString(), /document\.querySelectorAll\(\s*('|"|`)#utilities input\[\s*type\s*=\s*(('|")?)number\2\s*\]\s*\1\s*\)/);
+```
+
+You should assign the result of your `document.querySelectorAll()` call to `utilitiesInputs`.
```

Comment:
The instructions ask for the variable `utilitiesNumberInputs` but the test looks for `utilitiesInputs`.